hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2238,17 +2238,35 @@
   <property>
     <name>hdds.key.len</name>
     <value>2048</value>
-    <tag>SCM, HDDS, X509, SECURITY</tag>
+    <tag>SCM, HDDS, X509, SECURITY, CRYPTO_COMPLIANCE</tag>
+    <description>
+      SCM CA key length. This is an algorithm-specific metric, such as modulus
+      length, specified in number of bits.
+    </description>
+  </property>
+  <property>
+    <name>hdds.key.algo</name>
+    <value>RSA</value>
+    <tag>SCM, HDDS, X509, SECURITY, CRYPTO_COMPLIANCE</tag>
+    <description>
+      SCM CA key algorithm.
+    </description>
+  </property>
+  <property>
+    <name>hdds.security.provider</name>
+    <value>BC</value>
+    <tag>OZONE, HDDS, X509, SECURITY, CRYPTO_COMPLIANCE</tag>
     <description>
-      SCM CA key length.  This is an algorithm-specific metric, such as modulus length, specified in number of bits.
+      The main security provider used for various cryptographic algorithms.
     </description>
   </property>
```

Comment:
Could you highlight in the PR description that you also expose already existing configs to the ozone-default.xml with this patch? I'll do some other already existing config addition to the ozone-default.xml (with the `CRYPTO_COMPLIANCE` tag) in [HDDS-10732](https://issues.apache.org/jira/browse/HDDS-10732).

Reply:
Thank you @dombizita for the review, I've addressed your comments